infrastructure/server/oidc_clients_spar.tf

```diff
@@ -32,7 +32,7 @@ resource "aws_cognito_user_pool_client" "dev_spar_oidc_client" {
   }
 
   user_pool_id     = aws_cognito_user_pool.fam_user_pool.id
-  write_attributes = "${concat(var.minimum_write_list, ["custom:idp_display_name"])}"
+  write_attributes = "${concat(var.minimum_oidc_attribute_list, ["custom:idp_display_name"])}"
```

Comment:
the previous "var.minimum_write_list" includes "email", but after change to "var.minimum_oidc_attribute_list", it doesn't have email anymore

Reply:
See Conrad's comment above, looks like "email" becomes an optional option